### ISSUE TITLE
How do I run this?

### ISSUE THREAD

**Issue by 13thThief** [tobloef/neural-network#1]
Hello @tobloef !
I am interested in this ML library-free project and would like to run this (do I have your permission?)
But I do not know how to run this, all I have is Spyder in Windows. How do I go about that?
Thank You.

**Comment by tobloef** [comment]
The program requires the [NumPy](http://www.numpy.org/) module to run, which is maybe already installed as part of Spyder. I'm not that familiar with Spyder, but if you have NumPy installed you should just be able to download the code from this repository (click [here](https://github.com/tobloef/neural-network/archive/master.zip) if you don't know how) and run the main.py file.

And of course you have permissions to run it, that's why it put it up here in the first place. I hope you learn something and let me know if you need any more help.

**Comment by 13thThief** [comment]
Yes, spyder has everything it needs and the program (main.py) executed successfully with the output similar to : "Training data for # epochs, # learning rate, # batch size" "Epoch #1 done with 93% accuracy"

Is that all? Or am I missing some graphical output too?
